packages/modular-scripts/src/addPackage.ts

```diff
@@ -16,82 +16,138 @@ import LineFilterOutStream from './utils/LineFilterOutStream';
 import { ModularPackageJson } from './utils/isModularType';
 
 const packagesRoot = 'packages';
+const CUSTOM_TEMPLATE = '__CHOOSE_MY_OWN__';
 
 interface AddOptions {
-  destination: string;
-  type: string | void;
   name: string | void;
+  type: string | void;
```

Comment:
What's the benefit of using `string | void` instead of optionality (`?`) ?

Reply:
Good question. I think the difference is that the field must be present in the object, but can have an undefined value, rather than simply being missing from the object entirely.

If there's a benefit there, it's marginal. I copied the pattern that was already used rather than changing it though.